src/screens/GroupInfo/index.js

```diff
@@ -174,6 +174,10 @@ const GroupInfo = () => {
     handleMember();
   }, []);
 
+  React.useEffect(() => {
+    getMembersList();
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `useEffect` hook you added calls `getMembersList()` when the component mounts. However, it's not clear where this function comes from. Make sure that `getMembersList` is defined in the scope of your component or imported correctly.
```diff
  React.useEffect(() => {
-    getMembersList();
+    if (typeof getMembersList === 'function') {
+      getMembersList();
+    }
  }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->